core/src/test/java/org/elasticsearch/cluster/routing/allocation/AllocationCommandsTests.java

```diff
@@ -107,9 +121,10 @@ public void testAllocateCommand() {
                 .put(IndexMetaData.builder("test").settings(settings(Version.CURRENT)).numberOfShards(1).numberOfReplicas(1))
                 .build();
         RoutingTable routingTable = RoutingTable.builder()
-                .addAsNew(metaData.index("test"))
+                .addAsRecovery(metaData.index("test"))
```

Comment:
kk. cool. Can you add a comment?

> On 19 Jan 2016, at 11:42, Yannick Welsch <notifications@github.com> wrote:
> 
> In core/src/test/java/org/elasticsearch/cluster/routing/allocation/AllocationCommandsTests.java:
> 
> > @@ -107,9 +121,10 @@ public void testAllocateCommand() {
> >                  .put(IndexMetaData.builder("test").settings(settings(Version.CURRENT)).numberOfShards(1).numberOfReplicas(1))
> >                  .build();
> >          RoutingTable routingTable = RoutingTable.builder()
> > -                .addAsNew(metaData.index("test"))
> > +                .addAsRecovery(metaData.index("test"))
> 
> because later we test that allocating empty primary with accept_data_loss flag set to false fails.
> 
> —
> Reply to this email directly or view it on GitHub.
>